pkg/certifier/scorecard/scorecardRunner.go

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/checks"
+	"github.com/ossf/scorecard/v4/log"
+	sc "github.com/ossf/scorecard/v4/pkg"
+)
+
+func (s scorecardRunner) GetScore(repoName, commitSHA string) (*sc.ScorecardResult, error) {
+	defaultLogger := log.NewLogger(log.DefaultLevel)
```

Comment:
there should be a logger in the context with https://github.com/guacsec/guac/blob/main/pkg/logging/logger.go#L44, can probably initialize this in `scorecardRunner`

Reply:
I have to use `log.NewLogger()` because `checker.GetClients` takes that as an argument and can’t be replaced by `logging.FromContext()`.

Reply:
ah gotcha - can you add a comment here with this explanation. Thanks!

Reply:
👍